src/Modules/OrchardCore.Commerce.Payment.Exactly/Models/ChargeResponse.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Microsoft.Extensions.Localization;
+using OrchardCore.Commerce.MoneyDataType.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text.Json.Serialization;
+
+namespace OrchardCore.Commerce.Payment.Exactly.Models;
+
+public class ChargeResponse : IExactlyResponseData
+{
+    private static readonly Dictionary<string, ChargeResponseStatus> _statusMap = new()
+    {
+        ["action-required"] = ChargeResponseStatus.ActionRequired,
+        ["processing"] = ChargeResponseStatus.Processing,
+        ["processed"] = ChargeResponseStatus.Processed,
+        ["failed"] = ChargeResponseStatus.Failed,
+    };
+
+    public string Type => "charge";
+    public string Id { get; set; }
+    public ChargeAttributes Attributes { get; set; }
+
+    [SuppressMessage(
+        "StyleCop.CSharp.NamingRules",
+        "SA1313:Parameter names should begin with lower-case letter",
+        Justification = "Necessary for localization extractor.")]
+    public static IDictionary<string, LocalizedString> GetResultCodes(IStringLocalizer<ChargeResponse> T) =>
+        new Dictionary<string, LocalizedString>
+        {
+            ["success"] = T["processing of the transaction was successfully completed"],
```

Comment:
They are copied [from the documentation](https://exactly.com/docs/api#tag/Transactions/operation/getTransactionById) as-is. I don't really want to edit them because that will make future maintenance harder.
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/4007293/873011f9-d4f0-4179-aae7-a289aa5aefac)
Also they are only used like `"Your transaction has failed: {0}."` (e.g. "Your transaction has failed: transaction failed, no specific details could be provided.") so it works.